pom.xml

```diff
@@ -98,6 +98,8 @@
         <module>docs</module>
         <module>tests</module>
         <module>rest-client</module>
+        <module>multitenantcy/tenant-manager-datamodel</module>
```

Comment:
Typo in package name multitenantcy should by multitenancy.

Reply:
o my ...